google/resource_compute_url_map_test.go

```diff
@@ -129,126 +156,130 @@ func testAccCheckComputeUrlMapExists(n string) resource.TestCheckFunc {
 func testAccComputeUrlMap_basic1(bsName, hcName, umName string) string {
 	return fmt.Sprintf(`
 resource "google_compute_backend_service" "foobar" {
-    name = "urlmap-test-%s"
+    name          = "urlmap-test-%s"
     health_checks = ["${google_compute_http_health_check.zero.self_link}"]
 }
 
 resource "google_compute_http_health_check" "zero" {
-    name = "urlmap-test-%s"
-    request_path = "/"
+    name               = "urlmap-test-%s"
+    request_path       = "/"
     check_interval_sec = 1
-    timeout_sec = 1
+    timeout_sec        = 1
 }
 
 resource "google_compute_url_map" "foobar" {
-    name = "urlmap-test-%s"
-	default_service = "${google_compute_backend_service.foobar.self_link}"
+    name            = "urlmap-test-%s"
```

Comment:
Looks like a few of these have spaces instead of tabs, which is messing up the indentation in github